modules/statistics/php/NDB_Menu_statistics_site.class.inc

```diff
@@ -23,7 +23,16 @@
  */
 class NDB_Menu_Statistics_Site extends NDB_Menu
 {
-
+    /**
+     * Checking user's permission
+     *
+     * @return bool
+     */
+    function _hasAccess()
+    {
+        $user =& User::singleton();
+        return $user->hasPermission('data_entry');
+    }
```

Comment:
@driusan doesn't the _checkCriteria have the CenterID check?

`function _checkCriteria($centerID, $projectID)
    {
        if (!empty($centerID)) {
            $this->query_criteria   .= " AND s.CenterID =:cid ";
            $this->query_vars['cid'] = $centerID;
        }
        if (!empty($projectID)) {
            $this->query_criteria   .= " AND c.ProjectID =:pid ";
            $this->query_vars['pid'] = $projectID;
        }
    }`